scripts/ci/jobs/osd_aws_qa_e2e_tests.py

```diff
@@ -8,6 +8,7 @@
 from clusters import AutomationFlavorsCluster
 
 # set required test parameters
+os.environ["DEPLOY_STACKROX_VIA_OPERATOR"] = "true"
 os.environ["ORCHESTRATOR_FLAVOR"] = "openshift"
 os.environ["SENSOR_HELM_DEPLOY"] = "true"
 
```

Comment:
This only applies to the `.deploy` stuff so lets delete is.
```suggestion
```

Reply:
Good catch!